package.json

```diff
@@ -38,24 +35,21 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git@github.com:sveltestrap/sveltestrap.git"
+    "url": "https://github.com/sveltestrap/sveltestrap.git"
   },
   "files": [
     "dist",
     "src"
   ],
   "exports": {
     ".": {
-      "types": "./src/index.d.ts",
-      "svelte": "./src/index.js",
-      "require": "./dist/index.js",
-      "module": "./dist/index.ejs",
-      "import": "./dist/index.ejs"
+      "types": "./dist/index.d.ts",
+      "svelte": "./dist/index.js"
     }
   },
   "scripts": {
     "start": "storybook dev -p 6006",
-    "build": "vite build",
+    "build": "svelte-package --input ./src && publint --strict",
```

Comment:
I'm curious why you want to move to svelte-package over the previous bundling? This removes the compiled sources entirely, leaving only the original source, which might not work for everyone.

Reply:
I've found it to be a fantastic tool for Svelte library development! Since all Svelte apps need to go through a compile step, there shouldn't be a problem with shipping what is effectively preprocessed source. I consider it similar to how the Svelte team only ship source for Svelte and Kit. I've used it for `@tanstack/svelte-query` and with >300,000 downloads there haven't been any bundling/resolution issues reported.

Reply:
Thanks for clarifying. This definitely simplifies things, and will be adopted elsewhere.